test/integration/master_routes_test.go

```diff
@@ -99,6 +99,7 @@ var expectedIndex = []string{
 	// "/healthz/poststarthook/extensions/third-party-resources",  // Do not enable this controller, we do not support it
 	"/healthz/poststarthook/generic-apiserver-start-informers",
 	"/healthz/poststarthook/kube-apiserver-autoregistration",
+	"/healthz/poststarthook/oauth.openshift.io-EnsureBootstrapOAuthClients",
```

Comment:
why is this camelcase while the rest is lowecase?

Reply:
> why is this camelcase while the rest is lowecase?

we're working through what it should look like.